### DISCUSSION TITLE
Progress bar: Distinguishing between intermediary and end states

### DISCUSSION THREAD

**Discussion by diderich** [YetiForceCompany/YetiForceCRM#14960] [Feature improvement]
YetiForce allows to display the progress of e.g. an Opportunity, Project, and ServiceContract:
![aaaa](https://user-images.githubusercontent.com/10230144/125594985-3ec5af3c-7f4b-497b-a6aa-0c1786eb1826.jpg)

Some of the states, like Completed and Canceled are not sequential, but end states. They can be configured, e.g. in
![bbbb](https://user-images.githubusercontent.com/10230144/125595086-38988e5e-a924-4aa0-9609-cc4641e51039.jpg)

Sometimes the state shown is misleading as all states are assumed to form a sequence. In the example, the project is Canceled. Nevertheless, as the state Competed precedes the state Canceled, it is market as completed (check mark).

It would be great if, when a progress status enters an end stage (as configured), the other end states (as configured) would be marked by an (X), rather than a tick mark (as shown in the figure above, which is in the state of Canceled).

**Comment by s2b-git** [comment]
Observed similar... once you get to status "Response received" via automated ticket "statusing" and you manually change it to "In progress" any other consecutive for instance emails from ticket Account or Contact, or comments sent from ticket, will not change statuses anymore.
